media/js/respiration/species.js

```diff
@@ -77,10 +77,10 @@
     }
 
     function initSpecies() {
-        var leafSpecies = $('leaf-numspecies').value;
+        var leafSpecies = parseInt($('leaf-numspecies').value);
```

Comment:
leafSpecies was coming through as a string, "0". Adding the parseInt fixed the flow of code here.